NAMESPACE

```diff
@@ -128,6 +128,7 @@ export(UserApi)
 export(Writer)
 export(array_info)
 export(compute)
+export(compute_sequentially)
```

Comment:
Is this new export intentional?

Reply:
TL;DR yes

I did a rebase of several PRs; previously they were stacked; not so now; the function is old but the export is newly put here. Ideally would be on a separate PR.